docs/admin-ports.md

```diff
@@ -6,25 +6,24 @@ The following is a list of ports used by internal DC/OS services, and their corr
 
 ### TCP
 
+ - 8053: dcos-net
  - 61053: dcos-mesos-dns
  - 61420: dcos-epmd
- - 61421: dcos-minuteman
- - 62053: dcos-spartan
- - 62080: dcos-navstar
- - 62501: dcos-spartan
- - 62502: dcos-navstar
- - 62503: dcos-minuteman
+ - 62053: dcos-net
+ - 62080: dcos-net
```

Comment:
Maybe clarify each of the dcos-net ports - like `- 62053 dcos-net (dns)` or something similar?